### DISCUSSION TITLE
integration with .net core

### DISCUSSION THREAD

**Discussion by WPmeet** [Apicurio/apicurio-registry#2977] [Q&A] [answered]
Guys, I am looking for example/sample to use Apicurio with .net based clients, We are using confluent lib to connect to Kafka, 
 I am struggling to find anything on how to Serialiser or Deserialisers for Avro.

anyone has exp or Pointers .

thanks and any help is highly appreciated

**Comment by EricWittmann** [comment, marked as answer]
We don't really have much experience with .net integrations.  If you are using Confluent tools/libs to connect to Kafka, you could do the same for the SerDes layer (I assume Confluent has .net support but am not sure).  We have an API that is compatible with the Confluent Schema Registry API.  When running Apicurio Registry you can find it here:

http://domain.example.com/apis/ccompat/v6
